src/kirby/components/fab-sheet/fab-sheet.component.ts

```diff
@@ -20,6 +31,40 @@ export class FabSheetComponent implements AfterContentInit {
 
   @ContentChild(ActionSheetComponent, { static: false }) actionSheet: ActionSheetComponent;
 
+  @ViewChild(IonFabButton, { static: true, read: ElementRef }) ionFabButton: ElementRef<
+    HTMLElement
+  >;
+
+  ngAfterViewInit(): void {
+    const kirbyCloseIcon = kirbyIconSettings.icons.find((icon) => icon.name === 'close');
+    this.setCloseIcon(kirbyCloseIcon);
+  }
+
+  private setCloseIcon(kirbyCloseIcon: Icon, retryCount = 0) {
+    const maxRetryCount = 20;
+    const retryDelayInMs = 20;
+    const fabButtonElement = this.ionFabButton.nativeElement;
+    if (!fabButtonElement || !kirbyCloseIcon || retryCount >= maxRetryCount) {
+      return;
+    }
+    if (fabButtonElement.shadowRoot && fabButtonElement.shadowRoot.innerHTML) {
+      const closeIcon = fabButtonElement.shadowRoot.querySelector('.close-icon ion-icon');
+      if (closeIcon) {
+        const closeIconSvgLoaded = closeIcon.shadowRoot.querySelector('.icon-inner svg');
+        const ionCloseIcon = (closeIcon as unknown) as IonIcon;
+        if (ionCloseIcon && closeIconSvgLoaded) {
+          console.log(
```

Comment:
Either this should be considered removed, or only be logged out during !prod

Reply:
Good point, fixed 👍